Header.hpp

```diff
@@ -81,18 +83,18 @@ struct Mmpose_params{
     }
 };
 
-struct Size {
-    Q_GADGET
-    Q_PROPERTY(int width MEMBER width)
-    Q_PROPERTY(int height MEMBER height)
+// struct Size {
```

Comment:
you can remove the code entirely. good practice is to just use git as history of code that ws written and then removed

Reply:
will do